assembl/views/__init__.py

```diff
@@ -316,6 +317,7 @@ def process_export_list(ls):
         application_url=application_url,
         get_route=get_route,
         user=user,
+        loggedin_userid=authenticated_userid(request),
```

Comment:
Shouldn't it be `effectiveuserid` here instead?

Reply:
loggedin_userid is explicitly about knowing that the person is logged in even though they have no role in the discussion. Adding a comment.